exporter/awscloudwatchlogsexporter/factory.go

```diff
@@ -54,8 +54,10 @@ func createLogsExporter(_ context.Context, set component.ExporterCreateSettings,
 		exporterhelper.WithStart(exporter.Start),
 		exporterhelper.WithShutdown(exporter.Shutdown),
 		exporterhelper.WithQueue(exporterhelper.QueueSettings{
-			Enabled:      true,
-			NumConsumers: 1, // due to the sequence token, there can be only one request in flight
+			Enabled: true,
+			// due to the sequence token, there can be only one request in flight
+			NumConsumers: 1,
+			QueueSize:    oCfg.QueueSettings.QueueSize,
```

Comment:
You should set this in the `createDefaultConfig` and validate not overwritten

Reply:
Comment for the NumConsumers.

Reply:
I see what you are saying, but actually given that `oCfg.QueueSettings` has just the `QueueSize` field, `Enabled` and  `NumConsumers` cannot be overwritten.